bgbackup.sh

```diff
@@ -502,7 +504,7 @@ fi
 mysqlcreate
 
 # Check that mysql client can connect
-$mysqlcommand "SELECT 1 FROM DUAL"
+$mysqlcommand "SELECT 1 FROM DUAL" 1>/dev/null
```

Comment:
Look at line 508 just below. It's looking for that to equal 1. If it does, then it should be returning the error on line 509. To determine if it's exiting with a 1 (line 512) or just displaying what it is selecting, change it to "select 0 from dual" and test.